### ISSUE TITLE
Dynamic ordering of the tooltip figures for the line graph

### ISSUE THREAD

**Issue by dantravers** [openclimatefix/nowcasting#289]
Currently the tooltip is a fixed order (OCF 4 hr forecast, OCF Forecast, PV Live updated, PV Live initial estimate).

Now that we have up to 4 lines, it can be quite hard to see all the points on the graph.  When you mouse-over, it would be helpful for understanding the order of the lines if the tooltip was in order of the lines for the half-hour you are looking at..  basically sort by the MW figure in the tooltip. 

Is this possible?

**Comment by braddf** [comment]
@dantravers Revisiting this, not sure if is right UX-wise, but admin it is also potentially not ideal as is – I think we'll sort the P levels stuff first and then maybe get some user feedback along with other features/data in future.

**Comment by dantravers** [comment]
Yep, sounds good.  Might want to mock this up to have a look, as I agree - it could be strange.
